modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/CustomElementPortletEntryLocalServiceImpl.java

```diff
@@ -39,9 +64,172 @@
 public class CustomElementPortletEntryLocalServiceImpl
 	extends CustomElementPortletEntryLocalServiceBaseImpl {
 
-	/*
-	 * NOTE FOR DEVELOPERS:
-	 *
-	 * Never reference this class directly. Use <code>com.liferay.remote.app.service.CustomElementPortletEntryLocalService</code> via injection or a <code>org.osgi.util.tracker.ServiceTracker</code> or use <code>com.liferay.remote.app.service.CustomElementPortletEntryLocalServiceUtil</code>.
-	 */
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CustomElementPortletEntry addCustomElementPortletEntry(
+			long userId, Map<Locale, String> nameMap, String tagName,
+			ServiceContext serviceContext)
+		throws PortalException {
+
+		User user = userLocalService.getUser(userId);
```

Comment:
This does not use the burrito pattern, right? 

Of defining first the attributes that you use first.

Reply:
:thinking: It doesn't, but I copied it from other place, so it's strange... I will reorder it, in any case :shrug: